### PR TITLE
JsInfra: Create nav links to each personal page

### PR DESCRIPTION
**Pull request by HaoxinLuo**

### CODE REVIEW

File: js/src/components/Welcome/Welcome.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { Title, Text, Anchor } from '@mantine/core'
 import classes from './Welcome.module.css'
+import { RootRoutes } from '../../Routes'
```
Comment (arklian) [inline js/src/components/Welcome/Welcome.tsx:3]: We should really get aliasing working when we can.

**Review by arklian — APPROVED** [review]
Nice

**Comment by arklian** [comment]
Nit: This should likely be in like "JsInfra:" and not "Lucas:"

File: js/src/components/Welcome/Welcome.tsx (hunk 1 above)
Comment (HaoxinLuo) [reply js/src/components/Welcome/Welcome.tsx:3]: will do in a separate PR

File: js/src/components/Welcome/Welcome.tsx [hunk 2]
```diff
@@ -37,6 +39,14 @@ export function Welcome() {
         </Anchor>
         . To get started edit pages/Home.page.tsx file.
       </Text>
+      <ul>
```
Comment (arklian) [inline js/src/components/Welcome/Welcome.tsx:42]: nit: Mantine has a List and List.Item component. Not sure how it is though.

**Review by arklian — APPROVED** [review]
Nice!

**Pull request merged**